test/forward.test.cpp

```diff
@@ -5,6 +5,9 @@
 #include <Eigen/Core>
 using namespace Eigen;
 
+// For Special functions like Erf
+#include <unsupported/Eigen/SpecialFunctions>
```

Comment:
```suggestion
#include <eigen3/unsupported/Eigen/SpecialFunctions>
```

Reply:
@allanleal Just spotted: this include is no longer needed since the test using Eigen was removed